src/clj/game/core/access.clj

```diff
@@ -205,9 +204,10 @@
   [state side eid card]
   (trigger-event state side :pre-steal-cost card)
   (swap! state update-in [:stats :runner :access :cards] (fnil inc 0))
-  (let [cost (steal-cost state side card)
+  (let [additional-costs (steal-cost state side card)
+        cost (merge-costs (mapv first additional-costs))
```

Comment:
```suggestion
        cost (merge-costs (mapv value additional-costs))
```

Reply:
Same reasoning as above.

Reply:
This one is selecting the whole cost from the costs and sources list rather than pulling the costs themselves apart like the one above.

Reply:
Oh good call.